Modules/Core/Common/test/itkNumericTraitsTest.cxx

```diff
@@ -1298,5 +1298,21 @@ itkNumericTraitsTest(int, char *[])
   // CompileTime Checks IsComplex traits does not return
   CheckIsComplexTraits();
 
+  itkConceptMacro(NumericTraitsCheck1, (itk::Concept::HasNumericTraits<long>));
+  itkConceptMacro(NumericTraitsCheck2, (itk::Concept::HasNumericTraits<std::complex<float>>));
+  itkConceptMacro(NumericTraitsCheck3, (itk::Concept::HasNumericTraits<itk::Vector<float, 3>>));
```

Comment:
Instead of numbering the checks, you could place each check in its own scope, to make it easier to remove or add checks in-between existing checks (without the need to renumber them), while avoiding naming conflicts:

```
  {
    itkConceptMacro(NumericTraitsCheck, (itk::Concept::HasNumericTraits<long>));
  }
  {
    itkConceptMacro(NumericTraitsCheck, (itk::Concept::HasNumericTraits<std::complex<float>>));
  }
  {
    itkConceptMacro(NumericTraitsCheck, (itk::Concept::HasNumericTraits<itk::Vector<float, 3>>));
  }

```

Another option could be to use more meaningful concept names, like CheckLong, CheckStdComplex, CheckItkVector:
```
  itkConceptMacro(CheckLong, (itk::Concept::HasNumericTraits<long>));
  itkConceptMacro(CheckStdComplex, (itk::Concept::HasNumericTraits<std::complex<float>>));
  itkConceptMacro(CheckItkVector, (itk::Concept::HasNumericTraits<itk::Vector<float, 3>>));
```

Just my two cent!